docs/faq.rst

```diff
@@ -30,6 +30,10 @@ If you want to remove this warning, you need to add ``:orphan:`` (see `full docs
 
 .. _full docs: https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#file-wide-metadata
 
+.. note::
+
+   Version 0.5 automatically adds this metadata value to remove this warning message.
```

Comment:
I'd probably just note this in the changelog and remove this section now that this feature exists.

Reply:
OK. This will be added to the Changelog automatically by the script.

I removed the section.